node/src/reactor.rs

```diff
@@ -234,10 +237,42 @@ impl RunnerMetrics {
     /// Create and register new runner metrics.
     fn new(registry: &Registry) -> Result<Self, prometheus::Error> {
         let events = IntCounter::new("runner_events", "total event count")?;
+
+        // Create an event dispatch histogram, putting extra emphasis on the area between 1-10 us.
```

Comment:
That comment isn't accurate is it?  Looks more like we're fine-grained for 100 us to 1 ms.

Reply:
You're right :(

The code is correct, the comment is wrong; that area was specifically the intended high-res section.